optuna/storages/_cached_storage.py

```diff
@@ -165,6 +165,12 @@ def get_all_study_summaries(self) -> List[StudySummary]:
 
         return self._backend.get_all_study_summaries()
 
+    def get_url(self) -> Optional[str]:
+
+        if hasattr(self, "_backend"):
+            return self._backend.url
```

Comment:
A nit, but how about using the `get_url` for consistency?
```suggestion
            return self._backend.get_url()
```

Reply:
Ah, thank you. You're right. 🙇